app/src/main/java/org/wikipedia/analytics/metricsplatform/MetricsPlatform.kt

```diff
@@ -19,6 +20,7 @@ object MetricsPlatform {
         "WikipediaApp/" + BuildConfig.VERSION_NAME,
         "android",
         "app",
+        Build.MODEL + Build.DEVICE,
         WikipediaApp.instance.languageState.systemLanguageCode,
```

Comment:
@dbrant is this the right data to populate `device_family` in the agent data object?